traffic_config.proto

```diff
@@ -66,8 +66,10 @@ message Action {
   oneof action {
     string rpc_name = 4;
     string action_list_name = 5;
+    string activity_name = 6;
   }
   optional bool send_response_when_done = 10;
+  optional bool kill_all_action_lists_when_done = 11 [default = false];
```

Comment:
can we rename this cancel_traffic_when_done.

Reply:
Done